docs/validation.md

```diff
@@ -0,0 +1,28 @@
+# Validation
+
+This document describes the image validation process as employed by Connaisseur.
+
+Since Connaisseur is a Kubernetes AdmissionController it works on requests and has these requests, the Kubernetes API, the registry and and the notary to work with. First off, Connaisseur does some light checks on the semantics of the request it receives, e.g. whether it is capable of using the API version of the admission request.
```

Comment:
`and and`

Also, I find "works on" and "work with" in the same sentence a bit confusing.